OLEs/ole-121.adoc

```diff
@@ -0,0 +1,45 @@
+....
+  OLE: 121
+  Layer: Consensus
+  Title: Omni UTXO-Based Assets
+  Author: Sean Gilligan
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/OmniLayer/Documentation/blob/master/OLEs/ole-121.adoc
+  Status: Draft
+  Type: Consensus
+  Created: 2023-08-03
+  License: BSD-2-Clause
+....
+
+== Introduction
+
+=== Abstract
+
+This document proposes adding a mechanism for Omni Smart Properties to migrate to a UTXO-based on-chain representation. This feature would be activated per-token by the token owner with the confirmation of a newly defined transaction type. Once activated for a token, individual token holders would be able to convert from Address-based to UTXO-based by sending Address-based tokens to a defined address and receiving the converted assets as change outputs.
+
+The converted assets would be able to interoperate with other protocols that require UTXOs for operation, but will lose some Omni functionality such as participation in the existing Omni Decentralized Exchange, Send To Owners functionality, etc.
+
+=== Copyright
+
+This OLE is licensed under the 2-clause BSD license.
+
+=== Motivation
+
+Since the launch of the Omni Layer, Omni has used Base58 Bitcoin addresses for sending and receiving Omni properties. Since the Omni Layer uses the Bitcoin network and Bitcoin private keys to sign transactions, this was the straightforward solution. Unfortunately, since all existing wallets that support Omni allow sending Omni properties to Bitcoin addresses, it is possible for users to send Omni tokens to Bitcoin wallets that do not support Omni transactions. When users send Omni properties to addresses in wallets that do not support Omni Layer transactions, it can be difficult or impossible to recover the transferred Omni properties.
```

Comment:
I assume this is part of the placeholder as well.

Reply:
That's a very generous way of saying that. 🤣

Removed and replaced with "TBD" and force-pushed.